floodviz/static/js/gage_map.js

```diff
@@ -0,0 +1,86 @@
+document.addEventListener("DOMContentLoaded", function (event) {
```

Comment:
Consider adding JSHint directives such as /* global proj */ . Besides making an IDE using a JSHint plugin happy, it's clearer what this module depends on.